server/controllers/auth.js

```diff
@@ -98,12 +100,29 @@ const slackOauth = (req, res) => {
     });
 };
 
+const verifySlackRequest = async (headers, body) => {
+  const slackSigningSecret = process.env.SLACK_SIGNING_SECRET;
```

Comment:
Is this new process.env.SLACK_SIGNING_SECRET?

Reply:
Yes, and also verification token should be removed from app deployed in Heroku.